src/index.js

```diff
@@ -1,40 +1,84 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Take a Shower',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Cook today',
-    completed: false,
-    index: 10,
-  },
-  {
-    description: 'Buy groceries',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: ' Homework',
-    completed: false,
-    index: 8,
-  },
-  {
-    description: ' List Project',
-    completed: false,
-    index: 4,
-  },
-];
+/*
+ localStorage.setItem('tasks', JSON.stringify(tasks));
+*/
```

Comment:
Please don't leave any commented code on your codebase. It will keep your codebase neat and professional. 👌